modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordSettingsState.kt

```diff
@@ -39,11 +49,17 @@ open class MirrordSettingsState : PersistentStateComponent<MirrordSettingsState.
         var showRolloutsInSelection: Boolean? = null
         var disabledNotifications: Set<NotificationId>? = null
         var showUsageBanner: Boolean = true
+        var mirrordFeedbackCounter: Int = 0
 
         fun disableNotification(id: NotificationId) {
             disabledNotifications = disabledNotifications.orEmpty() + id
         }
 
+        fun tickFeedbackCounter(): Int {
+            mirrordFeedbackCounter += 1
+            return mirrordFeedbackCounter
+        }
+
```

Comment:
suggestion, we can probably just use a getter/setter here idiomatic to kotlin, when you access the property you always get the incremented value https://kotlinlang.org/docs/properties.html#getters-and-setters 

this is also fine but just suggesting

Reply:
I'm up for being idiomatic. Thanks for the link.